marmoset/imagecatalog/image.py

```diff
@@ -46,6 +46,8 @@ def metadata(self):
                 parsed_yaml.get("distribution_version", None)
             metadata['architecture'] = parsed_yaml.get("architecture", None)
             metadata['active'] = parsed_yaml.get("active", True)
+            metadata['internal'] = parsed_yaml.get("internal", True)
```

Comment:
I think this should either default to `False` or have a configurable default setting.
Otherwise all existing deployments will suddenly have `internal` only images if they don't add that property to their metadata files.

Reply:
updated it